src/ts/subtitleutils.ts

```diff
@@ -84,7 +85,7 @@ export class SubtitleSwitchHandler {
 
     const offListItem: ListItem = {
       key: SubtitleSwitchHandler.SUBTITLES_OFF_KEY,
-      label: 'off',
+      label: getLocalizedText(i18n.t('labels.off')),
```

Comment:
I think the call of `getLocalizedText` should be the responsibility of the "renderer", i.e. `Component.toDomElement`.